components/ClientLibraries.js

```diff
@@ -0,0 +1,65 @@
+const ClientLibraries = () => {
+  return (
+    <>
+      <h2>Client libraries</h2>
+      <section className="client-libs">
+        <a href="https://github.com/stripe/stripe-ruby" target="_blank">
+          <h3>
+            <img src="/images/ruby-logo.svg" />
+            <div>Ruby</div>
+          </h3>
+        </a>
+        <a href="https://github.com/stripe/stripe-node" target="_blank">
+          <h3>
+            <img src="/images/node-logo.svg" />
```

Comment:
In order to be accessible, these img tags should have alt set  - `alt="Node"`, etc